enterprise/frontend/src/embedding-sdk/cli/snippets/nextjs-page-snippet.ts

```diff
@@ -0,0 +1,30 @@
+import path from "path";
+
+import { checkIfUsingAppOrPagesRouter } from "../utils/nextjs-helpers";
+
+// Next.js page for /app/analytics-demo/page.tsx or /pages/analytics-demo.tsx
+export const getNextJsAnalyticsPageSnippet = async (componentPath: string) => {
```

Comment:
This is to generate the route file. It works for both app router and pages router.